### ISSUE TITLE
MMB not compatible with Dynare 6.0

### ISSUE THREAD

**Issue by chskcau** [IMFS-MMB/mmb-gui-electron#170]
MMB -- the latest version available for download on the main website, https://www.macromodelbase.com -- appears to not be compatible with Dynare 6.0; the latest version of Dynare I was able to confirm works is 5.5, the previous version of Dynare.

**To Reproduce**
1. Install MATLAB (R2024a was used), Dynare 6.0, and MMB as necessary, and run MMB.
2. Open settings and select Dynare 6.0.
3. Click on "Check compatibility".
4. An error message pops up saying "There was an error running the tests [...] Original error message: Process quit with exit code 1."

**Comment by AlexDece** [comment]
Yes, it is an issue with Dynare 6. The Dynare team adjusted the routine. 
We are working on bringing the MMB to Dynare 6.
In the meantime, please continue using any Dynare version <6.
Alex
